client/state/selectors/get-editor-url.js

```diff
@@ -9,7 +9,10 @@ import { getEditorPath } from 'state/ui/editor/selectors';
 
 export const getEditorUrl = ( state, siteId, postId = null, postType = 'post' ) => {
 	if ( 'gutenberg' === getSelectedEditor( state, siteId ) ) {
```

Comment:
Maybe it would be safer if we include the same check we do for showing the opt-in notice:

```
const isGutenbergAvailable = () => 
				isEnabled( 'gutenberg/opt-in' ) 
				&& ( isEnabled( 'gutenberg' ) || isCalypsoifyGutenbergEnabled( state, getSelectedSiteId( state ) ) );
if ( isGutenbergAvailable &&  'gutenberg' === getSelectedEditor( state, siteId ) ) {
		return getGutenbergEditorUrl( state, siteId, postId, postType );
}
```

Reply:
We'd be duplicating the checks though 🤔

Reply:
Actually, it'd be triplicated ([#](https://github.com/Automattic/wp-calypso/blob/37bcf65a47d9aec56d2aa75994ce8d41809fe5f9/client/post-editor/editor-gutenberg-opt-in-sidebar/index.jsx#L65), [#](https://github.com/Automattic/wp-calypso/blob/37bcf65a47d9aec56d2aa75994ce8d41809fe5f9/client/post-editor/editor-gutenberg-opt-in-notice/index.jsx#L63).

What if create a new selector?

Reply:
Well, those are different contexts though! 😄 

Anyway, it sounds good to me, and I'm opening a proper PR instead of this 👍